frontend/src/apiAccess/apiApps.js

```diff
@@ -323,6 +324,12 @@ export const apiReduxApps = {
     responseEntityType: "cards",
     responseIsList: false,
   }),
+  LIST_EVENT_TYPES: createReduxApp({
```

Comment:
most of the time the word "Fetch" is used. When in doubt, be consistent. If we each choose our own naming convention then things will become a bit unpredictable.

If you think your convention is better then it would be good to update the other things so they match.